git/operations.go

```diff
@@ -289,7 +293,18 @@ func execGitCmd(ctx context.Context, dir string, out io.Writer, args ...string)
 }
 
 func env() []string {
-	return []string{"GIT_TERMINAL_PROMPT=0"}
+	env := []string{"GIT_TERMINAL_PROMPT=0"}
+
+	// include allowed env vars from os
+	for _, k := range allowedEnvVars {
+		v, ex := os.LookupEnv(k)
+		if ex == false {
+			continue
+		}
+		env = append(env, k+"="+v)
+	}
```

Comment:
The golang idiomatic way to write this would be
```golang
if v, ok := os.LookupEnv(k); ok {
    env = append(env, k+"="+v)
}
```

Reply:
You are right and I should have thought about it as I was already familiar with it.